solr/core/src/java/org/apache/solr/core/SolrCore.java

```diff
@@ -1370,13 +1366,19 @@ public void initializeMetrics(SolrMetricsContext parentContext, Attributes attri
 
     newSearcherTimer =
         new AttributedLongTimer(
-            baseSearcherTimerMetric,
-            Attributes.builder().putAll(baseSearcherAttributes).put(TYPE_ATTR, "new").build());
```

Comment:
To me, it seemed less organized to differentiate "warmup" from "open" by an attribute, so I separated them.